### DISCUSSION TITLE
Method Not Allowed

### DISCUSSION THREAD

**Discussion by Zebi15** [filamentphp/filament#13817] [Help]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

I started a local filament project and developed. Today I hosted the platform on cpanel. I put the commands in the terminal:
php artisan migrate:fresh , php artisan route:cache, route:clear , php artisan filament:optimize-clear, etc.
I created a new user also from the terminal and then I tried to log in. The page has loaded, I filled in the data, but after clicking log in I receive the following error:
Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException
The POST method is not supported for route admin/login. Supported methods: GET, HEAD.

Locally I have the same code and I do not encounter that error,

.env file:
APP_NAME=Laravel
APP_ENV=production
APP_KEY=aaa
APP_DEBUG=true
APP_TIMEZONE=UTC
APP_URL=aaa

APP_LOCALE=en
APP_FALLBACK_LOCALE=en
APP_FAKER_LOCALE=en_US

APP_MAINTENANCE_DRIVER=file
# APP_MAINTENANCE_STORE=database

BCRYPT_ROUNDS=12

LOG_CHANNEL=stack
LOG_STACK=single
LOG_DEPRECATIONS_CHANNEL=null
LOG_LEVEL=debug

DB_CONNECTION=mysql
DB_HOST=localhost
DB_PORT=3306
DB_DATABASE=ueokbntq_swiss_collection_filament
DB_USERNAME=myusername
DB_PASSWORD=mypasseord

SESSION_DRIVER=database
SESSION_LIFETIME=120
SESSION_ENCRYPT=false
SESSION_PATH=/
SESSION_DOMAIN=null

BROADCAST_CONNECTION=log
FILESYSTEM_DISK=local
QUEUE_CONNECTION=database

CACHE_STORE=database
CACHE_PREFIX=

MEMCACHED_HOST=127.0.0.1

REDIS_CLIENT=phpredis
REDIS_HOST=redis
REDIS_PASSWORD=null
REDIS_PORT=6379

MAIL_MAILER=smtp
MAIL_HOST=mailpit
MAIL_PORT=1025
MAIL_USERNAME=null
MAIL_PASSWORD=null
MAIL_ENCRYPTION=null
MAIL_FROM_ADDRESS="hello@example.com"
MAIL_FROM_NAME="${APP_NAME}"

AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
AWS_DEFAULT_REGION=us-east-1
AWS_BUCKET=
AWS_USE_PATH_STYLE_ENDPOINT=false

VITE_APP_NAME="${APP_NAME}"

SCOUT_DRIVER=meilisearch
MEILISEARCH_HOST=http://meilisearch:7700

MEILISEARCH_NO_ANALYTICS=false

**Comment by martio** [comment]
@Zebi15 i have the same error...

**Reply by Zebi15** [reply]
In my case, I solved the problem in the following way. On the host, my server points to public_html. But the filament and laravel project is in public_html/public. I went to support with the hosts and they changed my root document to public_html/public and it worked. Check where your server is pointing from and change it to point from /public in the folder where the project is located.

**Comment by indrakoto** [comment]
I have the same error too.. vps, almalinux, php 8.3, filament3, mysql, webserver apache/httpd. Help

**Reply by Zebi15** [reply]
In my case, I solved the problem in the following way. On the host, my server points to public_html. But the filament and laravel project is in public_html/public. I went to support with the hosts and they changed my root document to public_html/public and it worked. Check where your server is pointing from and change it to point from /public in the folder where the project is located.

**Comment by martio** [comment]
The issue in my case was that the "Livewire" library was being loaded over "http" in the production environment, while the application itself was running on "https."

The solution was to add the following code in the `app/Providers/AppServiceProvider.php` file:

```
use Illuminate\Support\Facades\URL;

// Force HTTPS in production environment
URL::forceHttps(force: $this->app->isProduction());
```

**Reply by Zebi15** [reply]
Good

**Comment by enessvg** [comment]
The problem seems to be livewire. Have you published your livewire files?

**Reply by martio** [reply]
@enessvg I encountered this error due to the default loading of Livewire because I had the `ASSET_URL` set. So, I had to enforce HTTPS. Without setting this environment variable, everything should be fine....

**Reply by enessvg** [reply]
> @enessvg I encountered this error due to the default loading of Livewire because I had the `ASSET_URL` set. So, I had to enforce HTTPS. Without setting this environment variable, everything should be fine....

@martio I was able to solve this problem by publishing livewire files.

**Reply by enessvg** [reply]
@Zebi15

**Reply by Zebi15** [reply]
@enessvg yes, i published livewire files, i don t know if before or after the following solution : On the host, my server points to public_html. But the filament and laravel project is in public_html/public. I went to support with the hosts and they changed my root document to public_html/public and it worked.
